mkdocs.yml

```diff
@@ -425,12 +425,18 @@ plugins:
         - template_macros.yaml
       on_error_fail: true
   - htmlproofer:
-      enabled: !ENV [ENABLED_HTMLPROOFER, false]
-      raise_error: true
+      enabled: !ENV [ENABLED_HTMLPROOFER, true]
```

Comment:
since you modified the default value, you need to update README to let the users know the default is true and they can disable by making this variable to false
https://github.com/edgexfoundry/edgex-docs/blob/23802aa8d2c88af9b403c7f720f512ad1b39d1b7/README.md?plain=1#L56-L62

Reply:
Didn't realize the instrucations is already at README so update the mkdocs.yml back to false.